tools/cpp/BUILD

```diff
@@ -181,6 +196,13 @@ filegroup(
     ],
 )
 
+filegroup(
+    name = "every-file-x86_32_windows",
```

Comment:
This is the same as "every-file-x64_windows", and "compile-x86_32_windows" is the same as "compile-x64_windows".

Please remove the new 32-bit specific rules, rename the 64-bit ones to indicate that they are also used for 32-bit (e.g. "compile-x64-and-x86_32_windows"), and use the renamed rules instead the new ones.

Reply:
Can I use simply `every-file-windows` and `compile-windows`?

Reply:
Sure!